src/PrestaShopBundle/ApiPlatform/Normalizer/ObjectDenormalizer.php

```diff
@@ -26,17 +26,23 @@
 
 declare(strict_types=1);
 
-namespace PrestaShopBundle\ApiPlatform\Converters;
+namespace PrestaShopBundle\ApiPlatform\Normalizer;
 
-class StringToIntConverter implements ConverterInterface
-{
-    public function convert(mixed $value): int
-    {
-        return (int) $value;
-    }
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer as SfObjectNormalizer;
 
-    public function supports(string $type): bool
+/**
+ * Used to be automatic injected in PrestaShopBundle/ApiPlatform/DomainSerializer::normalizers with
+ * prestashop.api.denormalizers tagged services.
+ */
+class ObjectDenormalizer extends SfObjectNormalizer
+{
+    /**
+     * ObjectDenormalizer must be the last denormalizer as a fallback.
+     *
+     * @return int
+     */
+    public static function getDefaultPriority(): int
     {
-        return 'int' === $type;
+        return 1;
```

Comment:
You can even use a negative value to be sure, which would be safer because I'm sure if the default priority is 0 or 1 🤔